cms/core/models/mixins.py

```diff
@@ -98,3 +100,16 @@ def get_context(self, request: "HttpRequest", *args: Any, **kwargs: Any) -> dict
         context: dict = super().get_context(request, *args, **kwargs)  # type: ignore[misc]
         context["subpages"] = self.get_paginator_page(request)
         return context
+
+
+class NoTrailingSlashRoutablePageMixin(RoutablePageMixin):
+    """A mixin to remove trailing slashes from RoutablePage routes.
+    This ensures that the output of routablepageurl and reverse_subpage does not
+    include a trailing slash.
+    """
+
+    def reverse_subpage(self, name: str, args: list[str] | None = None, kwargs: dict[str, Any] | None = None) -> str:
+        result: str = super().reverse_subpage(name, args, kwargs)
+        if not WAGTAIL_APPEND_SLASH and result.endswith("/"):
+            result = result[:-1]
```

Comment:
question: everywhere else you're using `.rstrip("/")`, why not here?

Reply:
Thanks, it was an oversight on my behalf, changed in [a436b45](https://github.com/ONSdigital/dis-wagtail/pull/360/commits/a436b450ef78e29a66c26da38c009168599b2f67)

Reply:
Comment has been addressed! Resolving on behalf of Dan!